tools/src/test/java/org/apache/kafka/tools/TopicCommandIntegrationTest.java

```diff
@@ -85,653 +80,761 @@
 import static org.mockito.Mockito.spy;
 
 @Tag("integration")
-@SuppressWarnings("deprecation") // Added for Scala 2.12 compatibility for usages of JavaConverters
-public class TopicCommandIntegrationTest extends kafka.integration.KafkaServerTestHarness implements Logging, RackAwareTest {
+@ExtendWith(ClusterTestExtensions.class)
+public class TopicCommandIntegrationTest {
```

Comment:
@TaiJuWu Could you please merge `TopicCommandTest` and `TopicCommandIntegrationTest` into single one? `ClusterTest` and `Test` can be existent in the same class

Reply:
Will do this weekend, Thanks!

Reply:
Updated already, please take a look, thanks!